frame/composable-traits/src/lending/mod.rs

```diff
@@ -22,15 +36,17 @@ pub type BorrowAmountOf<T> = <T as DeFiEngine>::Balance;
 pub struct UpdateInput<LiquidationStrategyId> {
 	/// Collateral factor of market
 	pub collateral_factor: MoreThanOneFixedU128,
-	///  warn borrower when loan's collateral/debt ratio
-	///  given percentage short to be under collaterized
-	pub under_collaterized_warn_percent: Percent,
+	/// warn borrower when loan's collateral/debt ratio
+	/// given percentage short to be under collateralized
+	pub under_collateralized_warn_percent: Percent,
 	/// liquidation engine id
 	pub liquidators: Vec<LiquidationStrategyId>,
 	pub interest_rate_model: InterestRateModel,
 }
 
 /// input to create market extrinsic
```

Comment:
delete this comment?

also, can delete all comments, as per compliation it is eveident and we introduced repeated text, while that is already in type system.

Remove comments?